docs/validator/running-a-validator.md

```diff
@@ -0,0 +1,252 @@
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set node_key_file secrets/node_key.json
+```
+
+## 8. Set the genesis file (required)
+
+```bash
+gnoland genesis generate
+```
+
+## 9. Add a validator (required)
+
+```bash
+# check the secrets file generated in step (2)
+$ gnoland secrets get
+[Node P2P Info]
+Node ID:  g19d8x6tcr2eyup9e2zwp9ydprm98l76gp66tmd6
+
+[Validator Key Info]
+Address:     g1lnha5yem9dmj0yfzysfqsnvrm6j2ywshq83qdf
+Public Key:  gpub1pggj7ard9eg82cjtv4u52epjx56nzwgjyg9zpleysamt23ar025757uepld60xztnw7ujc3gwtjuy4pwv6z9sh4g284h3q
+
+[Last Validator Sign State Info]
+Height:  0
+Round:   0
+Step:    0
+
+# add the validator to the genesis file using the address and the public key in the Validator Key Info section
+$ gnoland genesis validator add -address g1lnha5yem9dmj0yfzysfqsnvrm6j2ywshq83qdf -name node01 -power 10 -pub-key gpub1pggj7ard9eg82cjtv4u52epjx56nzwgjyg9zpleysamt23ar025757uepld60xztnw7ujc3gwtjuy4pwv6z9sh4g284h3q
+```
+
+## 10. Start the chain
+
+```bash
+gnoland start
+```
+
+:::warning
+
+Slashing policies for downtimes, double signing or any manipulation using the validator membership will be introduced in the near future.
+
+Setting up the Sentry Node Architecture minimizes the risks of DDoS targetting the validator node. Configuring the `unconditional_peer_ids` is essential. Refer to [this issue](https://github.com/gnolang/gno/issues/1923) for further information. :::
+
+# Connect to an Existing Gno Chain
+
+## 1. Initialize the configurations (required)
+
+```bash
+gnoland config init
+```
+
+## 2. Initialize the secrets (required)
+
+```bash
+gnoland secrets init
+```
+
+## 3. Set a moniker (optional)
+
+```bash
+gnoland config set moniker node02
+```
+
+## 4. Set the rpc connection address (required for connecting with other nodes)
+
+```bash
+gnoland config set rpc.laddr "tcp://0.0.0.0:26657"
+```
+
+## 5. Set the validator private key (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set priv_validator_key_file secrets/priv_validator_key.json
+```
+
+## 6. Set the validator state (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set priv_validator_state_file secrets/priv_validator_state.json
+```
+
+## 7. Set the node key (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set node_key_file secrets/node_key.json
+```
+
+## 8. Obtain the genesis file of the chain to connect to
+
+:::info
+
+The genesis file will be [easily downloadable from GitHub](https://github.com/gnolang/gno/issues/1836#issuecomment-2049428623) in the future.
+
+For now, obtain the file by
+
+1. Sharing via scp or ftp
+2. Getting from `{chain_rpc:26657}/genesis` (might result in time-out error due to large file size)
+
+:::
+
+```bash
+## TODO: Add link to download the file from GitHub
+```
+
+## 9. Add the new validator to existing chain
```

Comment:
What do you mean by this step? I think we should drop it

Reply:
As our left [comment](https://github.com/gnolang/gno/blob/8a5c5ab762b991a604c14c46de745ae2b862464f/docs/gno-infrastructure/validators/running-a-validator.md?plain=1#L198), there will be guide when its available.